cmd/inspect/inspect.go

```diff
@@ -28,4 +28,5 @@ func init() {
 		Short: "Inspect policy rules",
 	}
 	InspectCmd.AddCommand(inspectPolicyCmd())
+	InspectCmd.AddCommand(inspectPolicyDataCmd())
```

Comment:
When I imagined this I was thinking it would be part of the same `ec inspect policy` command. I guess this works as well.

Reply:
Actually I flip-flopped between a few options here, since `ec inspect policy --data-source <sourceurl>` could also make sense, or even some kind of autodetection of the source "kind", but I decided on a separate command, partly to avoid handling the possibility of the user specifying both policy and data source urls in one `ec inspect` command.